source/_drafts/Video-js-7-3-Responsive-Layout-fill-mode-createLogger.md

```diff
@@ -0,0 +1,66 @@
+---
+title: 'Video.js 7.3: Responsive Layout, Fill Mode, createLogger'
+author:
+  name: Gary Katsevman
+  github: gkatsev
+tags:
+---
+
+Another month, another Video.js release: [v7.3.0][]. This release brings a long wanted feature: Responsive Layout.
```

Comment:
Should this link to the section headers below?

Reply:
It does. It's a markdown shorthand, if the text matches what you named the link, you don't have to double it up. You can verify that the link works over here: https://deploy-preview-32--videojs-blog.netlify.com/